### PR TITLE
[ko] Update translation

### PR DESCRIPTION
**Pull request by marocchino**
- Update basic funtions
- Do not use colon
- Unify words: arity, behaviour
- Other minor fixes

### CODE REVIEW

**Comment by marocchino** [comment]
review please. @Dalgona @devleoper @riseshia @malkoG

File: ko/lessons/basics/pipe-operator.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ lang: ko
 
 ## 소개
 
-프로그래밍을 하다보면 코드가 지저분해집니다. 함수의 호출이 따라가기 힘들 정도로 많이 중첩되어 지저분해질 수도 있습니다. 아래 중첩된 함수 호출을 생각해 봅시다.
+프로그래밍을 하다보면 코드가 지저분해질 수 있습니다. 함수의 호출이 따라가기 힘들 정도로 많이 중첩되어 지저분해질 정도로요. 아래 중첩된 함수 호출을 생각해 봅시다.
```
Comment (marocchino) [inline ko/lessons/basics/pipe-operator.md]: Original was:

> Programming can get messy. So messy in fact that function calls can get so embedded that they become very difficult to follow.

I don't think my change is better. but, `can get messy` is not `get messy`. If you have any suggestion, please comment.

Comment (riseshia) [inline ko/lessons/basics/pipe-operator.md]: 👍

File: ko/lessons/basics/functions.md [hunk 2]
```diff
@@ -93,9 +93,32 @@ iex> Length.of [1, 2, 3]
...
+
+iex> Greeter2.hello()
+"Hello, anonymous person!"
+iex> Greeter2.hello("Fred")
+"Hello, Fred"
+iex> Greeter2.hello("Fred", "Jane")
+"Hello, Fred and Jane"
+```
+
+함수 이름을 주석으로 달아두었습니다. 첫 번째 구현은 인자를 받지 않습니다. 그래서 `hello/0`라 합니다. 두 번째는 하나의 인자를 받고 `hello/1`라 하고 계속 이런 식입니다. 다른 언어의 함수 오버로드와는 다르게 3개의 서로 _다른_ 함수가 존재합니다.(아까 전에 설명했던 패턴매칭은 인자 개수가 같은 함수의 선언이 여러번 있을 때만 적용됩니다.)
```
Comment (riseshia) [inline ko/lessons/basics/functions.md]: `여러번` -> `여러 번`

File: ko/lessons/basics/functions.md [hunk 3]
```diff
@@ -161,9 +183,7 @@ iex> Greeter.hello("Sean", "es")
 "Hola, Sean"
 ```
 
-
-가드 예제에 디폴트 인자를 적용한 경우를 다뤄보도록 합시다. 아마, 다음과 같이 나타낼 수 있을 겁니다.
-
+가드 예제에 기본값 인자를 적용한 경우를 다뤄보도록 합시다. 아마, 다음과 같이 나타낼 수 있을 겁니다.
```
Comment (riseshia) [inline ko/lessons/basics/functions.md]: (maybe) `아마,` -> `아마`

File: ko/lessons/basics/functions.md [hunk 4]
```diff
@@ -184,9 +204,7 @@ end
 ** (CompileError) def hello/2 has default values and multiple clauses, define a function head with the defaults
 ```
 
-
-Elixir에서 여러 매칭 함수에 디폴트 인자가 들어가는 것을 권장하지 않습니다. 다소 헷갈릴 수 있기 때문입니다. 이를 다루기 위해서, 디폴트 인자가 들어있는 함수 선언문을 추가해봅시다.
-
+Elixir에서 여러 매칭 함수에 기본값 인자가 들어가는 것을 권장하지 않습니다. 혼동할 수 있기 때문입니다. 이를 다루기 위해서, 기본값 인자가 들어있는 함수 선언문을 추가해봅시다.
```
Comment (riseshia) [inline ko/lessons/basics/functions.md]: `Elixir에서` -> `Elixir는` or `Elixir에서는`

**Comment by riseshia** [comment]
I'm done XD

**Comment by malkoG** [comment]
Nice!

**Comment by devleoper** [comment]
Let's Get This Merged!

**Pull request merged**